app/views/shinycms/discussions/_comment_thread.html.erb

```diff
@@ -0,0 +1,14 @@
+<% if comment.hidden %>
```

Comment:
ActiveRecord generates `predicate?` methods for boolean columns like `hidden`, so line 1 could read (if you prefer): `if comment.hidden?`.